core-model/build.gradle.kts

```diff
@@ -22,4 +23,15 @@ kotlin {
             }
         }
     }
-}
\ No newline at end of file
+}
+
+dependencies {
```

Comment:
@takahirom I know this is questionable but I need a place where we could satisfy the direct exposure to the ios-combined module. Now the options are;

```
val projects = listOf(
            projects.coreModel,
            projects.coreDesignsystem,
            projects.coreData
        )
```

I found `coreModel` to be the sanest one.

Reply:
I don't dislike this idea. 👀 
But originally, I added core resources.
I have a question about this. Can we use string value in the model class?

https://github.com/DroidKaigi/conference-app-2022/pull/119/files#diff-c053860dd51d0a036f0a1ffc0adbca71dd1dcf8f9610e502da3e1c1f02db1f68R12

By the way, can you move it here?

```
kotlin {
    sourceSets {
        val commonMain by getting {
            dependencies {
// here

Reply:
Sure I can! 

I don't see a reason why we can't use the string value in the model class. It should be possible.